Computations/3DTiles/LyonTemporal/PythonCallingDocker/demo_strip_attributes.py

```diff
@@ -0,0 +1,125 @@
+import logging
+import os
+import sys
+import demo_configuration
+
+
+class StripInputsOutputs:
+    """
+    A utility class gathering the conventional names, relative to this demo,
+    used by the strip algorithms for designating its input/output directories
+    and filenames
+    """
+    def __init__(self,
+                 output_dir=demo_configuration.output_dir,
+                 vintages=demo_configuration.vintages,
+                 boroughs=demo_configuration.boroughs):
+        self.output_dir = output_dir
+        self.vintages = vintages
+        self.boroughs = boroughs
+
+    def get_output_dir(self, vintage, create=True):
+        """
+        :param vintage: a integer or string designating a year
+        :param create: when the proposed output directory does not already
+               exist and when create is True then create the output directory
+        :return: the proposed output directory of the strip attribute
+        """
+        if isinstance(vintage, int):
+            vintage = str(vintage)
+        strip_dir = 'LYON_' + vintage + '_Splitted_Stripped'
+        result_dir = os.path.join(self.output_dir, strip_dir)
+        if create and not os.path.isdir(result_dir):
+            logging.info(f'Creating strip output directory {result_dir}.')
+            os.mkdir(result_dir)
+        return result_dir
+
+    def get_resulting_files_basenames(self):
+        """
+        :return: the list of the basename (no directory) filenames that the
+                 strip algorithm is supposed to produce
+        """
+        result = list()
+        for vintage in self.vintages:
+            for borough in self.boroughs:
+                result.append(
+                    borough + '_BATI_' + str(vintage) + '_splitted_stripped.gml')
```

Comment:
This is providing the fact that the strip is called after the split. I would only keep the `derive_output_file_basename_from_input()` since it expresses a more generic case.